src/components/magicui/animated-grid-pattern.tsx

```
@@ -0,0 +1,154 @@
+
+export function AnimatedGridPattern({
+  width = 40,
+  height = 40,
+  x = -1,
+  y = -1,
+  strokeDasharray = 0,
+  numSquares = 50,
+  className,
+  maxOpacity = 0.5,
+  duration = 4,
+  repeatDelay = 0.5,
```

ESLint

[error] 'repeatDelay' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)